### DISCUSSION TITLE
Confluence rate limit & not republishing all

### DISCUSSION THREAD

**Discussion by sebastianaguerre** [bsorrentino/maven-confluence-plugin#277] [Q&A] [answered]
Hi, when the repository grows pretty big, with many pages, Confluence starts rejecting updates because of the limitation on how many calls can we do. This seems to happen because even with just a small change, the plugin attempts to republish everything again.

Is there any workaround or step I am missing to just try to publish in confluence the things I've just changed and not everything again?

**Comment by bsorrentino** [comment, marked as answer]
Hi @sebastianaguerre 

Thanks for feedback, since version `6.0.0` there is the [DeployState](http://bsorrentino.github.io/maven-confluence-plugin/deploy-mojo.html#deployState) that has been introduced specifically to avoid this issue.

Take a look and let me know
